### ISSUE TITLE
[$125] Color: Many Pages: Various links are identified only by use of color

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#76919]
## Action Performed:
1. Using Windows + Chrome, open the page https://new.expensify.com/
2. Observe the link 'Licenses', located in the footer, under the 'Get Started' section, and how it is indicated by color alone

## Expected Result:
Color is not the only visual means to distinguish links on the page. Each link on the page is visually identifiable with color and a second method (e.g underline, bold, italic, etc)

## Actual Result:
The link is identified only by use of color.

HTML:
<div dir="auto" class="css-146c3p1" style="color: rgb(175, 187, 176); font-size: 13px; text-align: center; font-family: &quot;Expensify Neue&quot;, &quot;Windows Segoe UI Emoji&quot;, &quot;Noto Color Emoji&quot;, &quot;Custom Emoji Font&quot;; font-style: normal; font-weight: 400; margin-bottom: 12px; margin-top: 8px;">Sign in with</div>

## Area issue was found in:
Landing Page

## Failed WCAG checkpoints
WCAG 1.4.1 Use of Color
https://www.w3.org/WAI/WCAG22/Understanding/use-of-color

## User impact:
Users with different disabilities employ assistive technology to adapt content according to their specific needs. For example, users who are blind (using a screen reader) benefit when information conveyed through color is also available in text (including text alternatives for images that use color to convey information), and users who are deaf-blind and use braille (text) refreshable displays may be unable to access color-dependent information.

## Suggested resolution:
Provide additional visual cues besides the link text color, e.g.: a permanent underline, bold or larger font, so that the button/link can be differentiated from other elements or surrounding text.
https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html

## Environment(s)
Windows Windows 10 - Chrome

## Screenshots/Videos
<img width="1918" height="997" alt="Image" src="https://github.com/user-attachments/assets/e314f415-feff-4555-9175-7e2d26839a8b" />

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6997265
**Issue reported by:** NA
**Slack conversation:** NA

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~022009685573946041834</li>
        <li>Upwork Job ID: 2009685573946041834</li>
        <li>Last Price Increase: 2026-01-09</li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @Pujan92</details>

**Comment by rushatgabhane** [comment]
the correct amount for this issue is $125

**Comment by mallenexpensify** [comment]
Putting on hold pending @dubielzyk-expensify 's comment above, before potentially closing

**Comment by rushatgabhane** [comment]
This issue was retested in **BFV cycle 544083 (Web)** on 2026-04-28 and is **Not Fixed**.

### Screenshots / Videos
- https://platform.applause.com/services/links/v1/external/1d500b9fa98ea78901863b67c6eeb136ac23c32d9ec74a0c8a30ae4ea72e1dff

### Retest notes
Not fixed: The link 'Licenses', located in the footer, under the 'Get Started' section, is identified only by use of color.
